### PR TITLE
created initial routes

### PR DESCRIPTION
**Pull request by sadatakhtar**
Created /auth/login - used to login with username and it returns the valid users name with a boolean  true.
Created /students/profile/:profile_id - used to get student profile data.
Created /student/tracker - used to get overall student progress data.

### CODE REVIEW

**Review by Wolfjawan — APPROVED** [review]
LGTM 👍

File: src/db/fakeData.js [hunk 1]
```diff
@@ -0,0 +1,147 @@
...
+    {
+        userName: "ekip",
+        pass: "cyf"
+    },
+    {
+        userName: "denis",
+        pass: "cyf"
+    }
+]
+ export const studentProfile = [
```
Comment (Denis-Andrei) [inline src/db/fakeData.js:15]: The data inside studentProfile should be organised a bit different. Each module should be an object that contains 3 key-value pairs, where the key will be each individual week and the value will be studentScore(initially 0)

File: src/db/fakeData.js [hunk 2]
```diff
@@ -0,0 +1,147 @@
...
+        },
+      pdSkills: {
+          skill: ["motivation", "communication", "confidence"],
+          skillLevel: ["Excellent", "Very good", "Average", "Below Average", "Poor"]
+      }
+    }
+    
+  ]
+  //route for /student-tracker
+   export const studentProgressTracker = [
```
Comment (Denis-Andrei) [inline src/db/fakeData.js:219]: studentProgressTracker should be an array with all the students Profile objects.

**Review by Denis-Andrei — CHANGES_REQUESTED** [review]

File: src/db/fakeData.js (hunk 1 above)
Comment (ISTANBULBEKLE) [reply src/db/fakeData.js:15]: The array we create is also reflecting what Clint's Regional Manager exposed us, I believe.

File: src/db/fakeData.js (hunk 2 above)
Comment (ISTANBULBEKLE) [reply src/db/fakeData.js:219]: For all the Student Tracker areas (homework, PD, attendance) would be taking from the same array/file?

**Review by ISTANBULBEKLE — APPROVED** [review]

File: src/db/fakeData.js (hunk 2 above)
Comment (sadatakhtar) [reply src/db/fakeData.js:219]: I have altered the data as requested, Can you please review

Comment (ISTANBULBEKLE) [reply src/db/fakeData.js:219]: Great guys, I think this will cover all the data request from the React.

**Review by ISTANBULBEKLE — APPROVED** [review]
Guys, great, thank you all.

**Review by Denis-Andrei — APPROVED** [review]

**Review by Denis-Andrei — APPROVED** [review]

**Review by osagiestar — COMMENTED** [review]
Everything is perfect, looks great. The coding is professionally written and nicely commented. Great job!

**Review by osagiestar — COMMENTED** [review]
Everything is perfect, looks great. The coding is professionally written and nicely commented. Great job!

**Comment by ISTANBULBEKLE** [comment]
Guys, please continue to do the merging, when reviews are done and there is no conflict. I did for some PRs but now on please when you see that reviews are done of course without any conflict just continue merging.

**Pull request merged**